Doxyfile

```diff
@@ -543,7 +559,7 @@ INTERNAL_DOCS          = NO
 # names in lower-case letters. If set to YES, upper-case letters are also
 # allowed. This is useful if you have classes or files whose names only differ
 # in case and if your file system supports case sensitive file names. Windows
-# and Mac users are advised to set this option to NO.
+# (including Cygwin) ands Mac users are advised to set this option to NO.
```

Comment:
Is this our typo or doxygen's?

```suggestion
# (including Cygwin) and Mac users are advised to set this option to NO.
```

Reply:
Probably an artifact of merging Doxyfile.template